### ISSUE TITLE
Modify Rubocop config

### ISSUE THREAD

**Issue by vheuken** [thibaudgg/video_info#139]
```
Style/DotPosition:
  Description: Checks the position of the dot in multi-line method calls.
  StyleGuide: https://github.com/bbatsov/ruby-style-guide#consistent-multi-line-chains
  Enabled: true
  EnforcedStyle: leading
  SupportedStyles:
  - leading
  - trailing
```

As for formatting specs, we should aim for something like:
```ruby
    it 'should apply width and height attributes properly' do
      dimensions = { width: 800, height: 600 }
      embed_code = subject.embed_code(iframe_attributes: dimensions)
      expect(embed_code).to match(/width="800"/)
      expect(embed_code).to match(/height="600"/)
    end
```

**Comment by vheuken** [comment]
@thibaudgg It looks like we already have that exact config in .rubocop.yml, so I'll try changing up the spec format again.

**Comment by vheuken** [comment]
Fixed in #140
